py3status/modules/check_tcp.py

```diff
@@ -3,13 +3,15 @@
 Display status of a TCP port on a given host.
 
 Configuration parameters:
-    cache_timeout: how often to run the check (default 10)
-    format: what to display on the bar (default '{host}:{port} {state}')
-    host: check if tcp port on host is up (default 'localhost')
-    port: the tcp port (default 22)
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default '{host}:{port} {state}')
+    host: check host (default 'localhost')
+    port: check port number (default 22)
+    string_down: show string when available (default 'UP')
+    string_up: show string when unavailable (default 'DOWN')
```

Comment:
I really don't like `string_..` being used in this way, because it is used as a placeholder replacement so `icon_..` or maybe `state_..` but not string at least yet.  Also `string_..` is just so general.

We do need to find a solution to these issues at some point